Fixed #34481 -- Added system check for reverse related fields in ModelAdmin.list_display. (#16757)
django/django: docs/releases/5.0.txt

:mod:`django.contrib.admin`
~~~~~~~~~~~~~~~~~~~~~~~~~~~

> * The ``admin.E109`` error now check not only many-to-many field,
>   it now also check reverse foreign key field.

Reviewer: The grammar here is incorrect; additionally the order of the sentence should be restructured to make it flow better:

```suggestion
* The ``admin.E109`` error now checks reverse foreign key attributes in
  addition to many-to-many fields.
```

Also corrected the term field to be attribute as it's not a model field - although nessita and felixx will clarify and adjust as necessary.

Reviewer 2: We don't add bugfixes and changes in system checks to release notes for a feature release. Please remove it.

Author: done
